.ci/setup.sh

```diff
@@ -57,11 +57,16 @@ install_docker() {
 
 enable_nested_virtualization() {
 	case "$arch" in
-	x86_64)
+	x86_64 | s390x)
 		if [ "$CI" == true ] && grep -q "N" /sys/module/kvm_intel/parameters/nested 2>/dev/null; then
```

Comment:
given you are doing a non-intel modprobe below, is this `kvm_intel` enabled check also not valid for s390x ?

Reply:
Yes it isn't. I overlooked that one.